test/functional-api/preferences/hub-preferences.it-spec.ts

```diff
@@ -61,13 +76,135 @@ beforeAll(async () => {
     HubPreferenceType.JOIN_HUB_FROM_HOST_ORGANIZATION_MEMBERS,
     'false'
   );
+  await changePreferenceHub(
+    entitiesId.hubId,
+    HubPreferenceType.ALLOW_MEMBERS_TO_CREATE_CHALLENGES,
+    'false'
+  );
 });
 
 afterAll(async () => {
   await removeHub(entitiesId.hubId);
   await deleteOrganization(entitiesId.organizationId);
 });
 
+describe('Hub Preferences - member create challenge preference', () => {
+  beforeAll(async () => {
+    await changePreferenceHub(
+      entitiesId.hubId,
+      HubPreferenceType.ALLOW_MEMBERS_TO_CREATE_CHALLENGES,
+      'true'
+    );
+  });
+
+  afterAll(async () => {
+    await changePreferenceHub(
+      entitiesId.hubId,
+      HubPreferenceType.ALLOW_MEMBERS_TO_CREATE_CHALLENGES,
+      'true'
```

Comment:
if you are setting it as you need in the 'before', do you really need to change it back? The same goes for everything else -if it's not a delete procedure, do you need to change the data as it was?

Reply:
Yes, it is better to have preferences states in specific  state in order, to be able to foresee the behavior.